eland/operations.py

```diff
@@ -947,7 +984,7 @@ def aggs_groupby(
     @staticmethod
     def bucket_generator(
         query_compiler: "QueryCompiler", body: "Query"
-    ) -> Generator[List[str], None, List[str]]:
+    ) -> Generator[Sequence[Any], None, Sequence[Any]]:
```

Comment:
We should try to keep this type `Generator[Sequence[str], None, Sequence[str]]`

Reply:
```
[{'key': {'groupby_Cancelled': False}, 'doc_count': 11381, 'sum_dayOfWeek': {'value': 32242.0}, 'sum_FlightDelayMin': {'value': 536325.0}, 'sum_AvgTicketPrice': {'value': 7162656.040748596}}, {'key': {'groupby_Cancelled': True}, 'doc_count': 1678, 'sum_dayOfWeek': {'value': 4793.0}, 'sum_FlightDelayMin': {'value': 81825.0}, 'sum_AvgTicketPrice': {'value': 1041708.8814849854}}]
```
So actually its `Sequence[Dict[str, Any]]`
When I initially wrote it , type wasn't correct